app/amity.py

```diff
@@ -0,0 +1,466 @@
+"""app/amity"""
+from random import choice
+import sys
+import os
+import subprocess
+import time
+import sqlite3
+
+from app.db.database import Base, Employees, Allocations, Rooms, DatabaseCreator, Unallocated
+from sqlalchemy import select
+
+
+class Amity(object):
+    def __init__(self):
+        self.all_people = []
+        self.all_fellow = ["h n"]
+        self.all_staff = ["j n"]
+        self.office_waiting_list = ["f f", "f g"]  # all staff without office
+        self.living_space_waiting_list = ["s g", "g m"]  # all fellow without living
+        self.all_rooms = ["VALHALLA", "JAVA", "PHP"]
+        self.all_offices = ["VALHALLA"]
+        self.all_living = ["JAVA", "PHP"]
+        self.room_allocations = {
+            "office": {
+                "VALHALLA": ["J M"]},
+            "living": {
+                "JAVA": [], "PHP": []}}
+
+    def create_room(self, room_name, room_type):
+        room_type = room_type.upper()
+        room_name = room_name.upper()
+        # check if room name exists
+        if room_name in self.all_rooms:
+            msg = ("%s - room name already used" % room_name)
+            print(msg)
+            # return(msg)
+        else:
+            # check if room type is correct
+            if(room_type in ('OFFICE', 'LIVING')):
+                self.all_rooms.append(room_name)
+
+                if room_type == "OFFICE":
+                    self.all_offices.append(room_name)
+                    self.room_allocations["office"][room_name] = []
+                    msg = ("%s - office added successfully" % room_name)
+                    print(msg)
+                    # return(msg)
+
+                elif room_type == 'LIVING':
+                    self.all_living.append(room_name)
+                    self.room_allocations["living"][room_name] = []
+                    msg = ("%s - living space added successfully" % room_name)
+                    print(msg)
+
+            else:
+                print("%s is a wrong room type" % room_type)
+
+    def add_person(self, first_name, last_name, role, accomodation="N"):
+        full_name = (first_name + " " + last_name).upper()
+        role = role.upper()
+        accomodation = accomodation
+
+        if full_name in self.all_people:
+            return ("%s already used " % full_name)
+        else:
+            if(role in ("FELLOW", "STAFF")):
```

Comment:
`if role in ("FELLOW", "STAFF"):` - The outer brackets are not necessary in Python